{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -44,7 +47,7 @@ yaqd-{{ cookiecutter.first_daemon_kind }} = "{{ cookiecutter.project_src_dir }}.
 
 [tool.black]
 line-length = 99
-target-version = ['py37', 'py38']
+target-version = ['py11']
```

Comment:
We may want to keep the oldest supported in here to avoid applying backwards incompatible syntax changes

I'd check black's recommendation here